source/manual/connect-to-vcloud-director.html.md

```diff
@@ -4,67 +4,78 @@ title: Connect to vCloud Director (Carrenza only)
 section: Infrastructure
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-08-08
+last_reviewed_on: 2019-11-26
 review_in: 6 months
 ---
 
-vCloud Director is the interface we use to manage our infrastructure in Carrenza.
-This includes virtual machines, gateways, firewalls and VPNs between providers.
+vCloud Director is the interface we use to manage our infrastructure in
+Carrenza. This includes virtual machines, gateways, firewalls and VPNs between
+providers.
 
 To access vCloud Director, you will need to connect to a Carrenza-provided VPN.
 You can use either Cisco AnyConnect or OpenConnect as a VPN client for this.
 
 ## Setting up the Cisco AnyConnect VPN profile on a Mac
 
-1. Clone the govuk-secrets private Git repo. The repo is very large but there is no need to download the full history.
+1. Clone the govuk-secrets private Git repo. The repo is very large but there
+   is no need to download the full history.
 
-    ```
-    mkdir -p ~/govuk && cd ~/govuk
-    git clone --depth 10 git@github.com:alphagov/govuk-secrets.git
+    ```sh
+    $ mkdir -p ~/govuk && cd ~/govuk
```

Comment:
Adding the $ makes it a lot harder to copy/paste: can we not?

Reply:
Without the `$` I found it hard to read because we've got the command to run and the output from the command in the same code block. I also wanted to discourage blindly copying commands without understanding what they're doing.

As it is, I'm not even sure we need a whole section about cloning `govuk-secrets` but I couldn't find a suitable page to link to.

Reply:
Fair enough. There's [the govuk-secrets pass README](https://github.com/alphagov/govuk-secrets/tree/master/pass)?

Reply:
I've amended the commit to link to the `README` and not include any code examples on how to clone it.